tests/test_sealed_box.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2013 Donald Stufft and individual contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+import binascii
+import os
+
+import pytest
+
+from nacl.encoding import HexEncoder
+from nacl.public import PrivateKey, PublicKey, SealedBox
+
+
+def _read_crypto_test_vectors(fname, maxels=0):
+    vectors = []
+    path = os.path.join(os.path.dirname(__file__), "data", fname)
+    with open(path, "rb") as fp:
+        for line in fp:
+            line = line.rstrip()
+            if line[0] == b'#'[0] or not line:
+                continue
+            splt = [x for x in line.split(b'\t')]
+            if maxels:
```

Comment:
It's an almost verbatim copy of `_read_crypto_test_vectors` in `test_generichash`; I just thought it would be better to postpone the refactor /unification to a single version into tests/utile.py to a later, single purpose PR.